app/views/settings/_form.html.erb

```diff
@@ -15,11 +15,11 @@
   </div>
   <div class="form-field">
     <%= f.label :accepted_threshold %>
-    <%= f.number_field :accepted_threshold %>
+    <%= f.number_field :accepted_threshold, step: 0.01 %>
```

Comment:
You can verify it with Ania, but I think it would be more convenient to have bigger step